### DISCUSSION TITLE
Unselect all entities with a single click (History graph)

### DISCUSSION THREAD

**Discussion by Youpimatin** [home-assistant/frontend#7392] [Dashboard]
### The request
It would be nice if you could select/unselect all entities with a single click in 'history' page.

![History](https://user-images.githubusercontent.com/55438591/98100160-c2f6a500-1e90-11eb-8625-fa665938de32.png)

### The alternative
Currently, when dealing with 55 entities with the same unit_of_measurement, you have to click 54 times to unselect all but the one you want to view in the graph. (No, you're not fool enough to check that there is indeed 55 entities on my screenshot!)

### Additional information
Maybe you could toggle selected/unselected entities by clicking on the unit_of_measurement text ? or add a small button ?

One button... to rule them all, one button to find them, one button to bring them all and in the darkness bind them.

**Comment by Youpimatin** [comment]
I didn't see this issue at the time :
**Provide method to hide all lines on a history chart** #6422

Currently, when dealing with 64 "%" entities with the same unit_of_measurement, you have to click 63 times to unselect all but the one you want to view in the graph :

2021.6:
![HistoryOld](https://user-images.githubusercontent.com/55438591/127479097-1491d5c8-c88d-4ef0-98eb-904b724251de.png)

2021.7:
![HistoryNew](https://user-images.githubusercontent.com/55438591/127479125-c196f864-9cdb-419d-a271-c33c4bf74da6.png)

**Comment by bradyjoh** [comment]
This is still an issue as far as I'm aware. Would be great to mirror the behavior of Grafana. It's the inverse of how HASS works. Clicking on a Name in the legend/table will "solo" it and show the line for that label, and you can hold down CTRL/CMD to highlight mutiple. Click a highlighted label again and it goes back to showing all. 

<img width="722" alt="image" src="https://github.com/user-attachments/assets/725ef95c-3a0e-43ef-ae58-81d8541dac4e" />
